### PR TITLE
Receive page

### PR DESCRIPTION
**Pull request by filiptronicek**
Last crucial page of the web app: the page for retrieving existing clips.

### CODE REVIEW

**Comment by filiptronicek** [comment]
@bigint if you could please see for yourself and review 🙏

File: src/pages/receive.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+import { Layout } from '../components/Layout';
+
+const sampleURLs = [
+  'https://interclip.app',
+  'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
+  'https://www.histories.cc/',
+  'https://devparty.io/',
+  'https://opensea.io/assets/matic/0x2953399124f0cbb46d2cbacd8a89cf0599974963/106565623496713384605063112493700845981575250044632175243042012644862217682945',
+  'https://www.npmjs.com/package/bruh-cli',
```
Comment (bigint) [inline src/pages/receive.tsx:19]: lint it

**Review by bigint — APPROVED** [review]
Rest LGTM 🚀

File: src/pages/receive.tsx (hunk 1 above)
Comment (filiptronicek) [reply src/pages/receive.tsx]: yessir

**Pull request merged**